Viewer query integration tests (#607)
kyma-incubator/compass: tests/end-to-end/e2e/README.md

# Compass end to end tests

> ## Test Scenario:

Reviewer: ```suggestion
Compass end to end test scenario consists of the following steps:
```

> - Get Dex token, create IntegrationSystem with it and request client credentials for IntSystem

Reviewer: ```suggestion
- Get a Dex token, create the Integration System using this token, and request client credentials for the IntegrationSystem
```

- Call Hydra for OAuth 2.0 access token with client_id and client_secret pair - https://github.com/kyma-incubator/examples/tree/master/ory-hydra/scenarios/client-credentials

> - Register an Application (as Integration System)

Reviewer: ```suggestion
- Register an Application as the Integration System
```

| USER_PASSWORD |    Dex static user password   |   - |
| DEFAULT_TENANT | Default tenant value |    `3e64ebae-38b5-46a0-b1ed-9ccee153a0ae` |
| DOMAIN | Kyma domain name |    `kyma.local` |

> | GATEWAY_JWTSUBDOMAIN | Default gateway for handling requests with JWT | compass-gateway |

Reviewer: ```suggestion
| GATEWAY_JWTSUBDOMAIN | Default gateway for handling requests with a JWT | compass-gateway |
```

> | GATEWAY_CLIENT_CERTS_SUBDOMAIN | Default gateway for handling requests with certificate | compass-gateway-mtls |

Reviewer: ```suggestion
| GATEWAY_CLIENT_CERTS_SUBDOMAIN | Default gateway for handling requests with a certificate | compass-gateway-mtls |
```

> | GATEWAY_OAUTH20_SUBDOMAIN | Default gateway forhandling requests with OAuth access token | compass-gateway-auth-oauth|

Reviewer: ```suggestion
| GATEWAY_OAUTH20_SUBDOMAIN | Default gateway for handling requests with an OAuth access token | compass-gateway-auth-oauth|
```
